DotaStats/PlayersModule/Presenter/PlayersModulePresenter.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+protocol PlayersModuleInput: AnyObject {
+    func showLoading()
+    func showPlayers(_ players: [Player])
+}
+
+protocol PlayersModuleOutput: AnyObject {
+    var view: PlayersModuleInput? { get set }
+    init(networkService: NetworkSeviceProtocol)
+    func players() -> [Player]
+    func playerByName(_ name: String) -> [Player]
+    func didSelectPlayer(_ player: Player)
```

Comment:
Подобные методы лучше писать приблизительно так
```suggestion
    func playersModule(_ module: PLayersModuleInput, didSelectPlayer player: Player)
```